tests/v2/validation/provisioning/k3s/provisioning_node_driver_test.go

```diff
@@ -110,7 +110,7 @@ func (k *K3SNodeDriverProvisioningTestSuite) TestProvisioningK3SClusterDynamicIn
 		client *rancher.Client
 	}{
 		{provisioninginput.AdminClientName.String(), k.client},
-		{provisioninginput.StandardClientName.String(), k.standardUserClient},
+		// {provisioninginput.StandardClientName.String(), k.standardUserClient},
```

Comment:
I think this one is commented cause sanity checks

Reply:
ah yes, let me fix it. Thanks